typescript/big-practice/src/scripts/utils/handleError.ts

```diff
@@ -3,6 +3,8 @@ import EventHelpers from '../helpers/eventHelpers';
 import renderConfirmPopup from './confirmPopup';
```

Comment:
You are including handling DOM and API errors in one. It will make it confusing and complicated to maintain code. You should split it into 2 files, one for DOM error and one for API error.